google/cloud/ndb/model.py

```diff
@@ -696,7 +696,7 @@ def _entity_from_protobuf(protobuf):
     return _entity_from_ds_entity(ds_entity)
 
 
-def _properties_of(entity):
+def _properties_of(*entities):
```

Comment:
The docstring below should change to match, and should specify that all the entities must be of the same type.

Reply:
97ac16e